core/server/utils/labs.js

```diff
@@ -1,9 +1,9 @@
-var config = require('../config'),
+var settingsCache = require('../api/settings').cache,
     flagIsSet;
 
+// @TODO: what is this lib doing?
```

Comment:
It is a util for when we need to detect if a flag is set before doing some logic. It was used for internal tags, and it lives as we may need it for some other feature in future.

There's also a whole heap of logic in subscribers and public API stuff that should also be split out to live in this util - stuff for conditionally registering helpers and middleware.